core/src/avm2/globals/flash/display/graphics.rs

```diff
@@ -1427,22 +1442,8 @@ fn handle_graphics_triangle_path<'gc>(
 
             let indices = &mut indices.iter();
 
-            while let Some((a, b, c)) = next_triangle(&vertices, indices, activation)? {
-                match culling {
-                    TriangleCulling::None => {
-                        drawing.draw_command(DrawCommand::MoveTo(a));
-
-                        drawing.draw_command(DrawCommand::LineTo(b));
-                        drawing.draw_command(DrawCommand::LineTo(c));
-                        drawing.draw_command(DrawCommand::LineTo(a));
```

Comment:
Where did these `draw_command` calls go? 🤔

Reply:
Into `draw_triangle_internal` :D!

Reply:
The code was exactly the same as in draw_triangle_internal so I just switched it over. I can switch back if that is preferred.

Reply:
Ah, right, I see! No need to switch back, it's nice this was. Just wasn't immediately obvious for me reading the diff.